lib/logstash/inputs/jdbc.rb

```diff
@@ -81,6 +83,9 @@ def register
     elsif File.exists?(@last_run_metadata_path)
       @sql_last_start = YAML.load(File.read(@last_run_metadata_path))
     end
+
+    # load statement from file, if filepath passed.
+    @statement = File.read(@statement) if File.exists?(@statement)
```

Comment:
can we add an explicit `statement_file` config to take the configuration from a SQL file? Much cleaner IMO